Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1299,12 +1287,17 @@ TextureCacheBase::GetXFBFromCache(const TextureLookupInformation& tex_info)
   return nullptr;
 }
 
-bool TextureCacheBase::LoadTextureFromOverlappingTextures(TCacheEntry* entry_to_update,
-                                                          const TextureLookupInformation& tex_info)
+TextureCacheBase::TCacheEntry*
+TextureCacheBase::GetTextureFromOverlappingTextures(const TextureLookupInformation& tex_info)
 {
-  bool updated_entry = false;
+  u32 numBlocksX = tex_info.native_width / tex_info.block_width;
 
-  u32 numBlocksX = entry_to_update->native_width / tex_info.block_width;
+  // XFBs created for the purpose of being a container for textures from memory
```

Comment:
Probably best to remove or rework this comment, it doesn't really make sense in this context